docs/core/tools/dotnet-restore.md

```diff
@@ -43,7 +43,23 @@ For dependencies, you specify where the restored packages are placed during the
 
 For project-specific tooling, `dotnet restore` first restores the package in which the tool is packed, and then proceeds to restore the tool's dependencies as specified in its project file.
 
-The behavior of the `dotnet restore` command is affected by some of the settings in the *Nuget.Config* file, if present. For example, setting the `globalPackagesFolder` in *NuGet.Config* places the restored NuGet packages in the specified folder. This is an alternative to specifying the `--packages` option on the `dotnet restore` command. For more information, see the [NuGet.Config reference](/nuget/schema/nuget-config-file).
+### Nuget.Config differences
```

Comment:
I see the NuGet doc use different casing for this file. Is the file really capitalized like this or should it be nuget.config?

Reply:
Good catch. I didn't notice that, I think it should be `nuget.config` and no other weird casing. I think I was undoing the G in NuGet and didn't notice the C and really the N at the start.